cucumber/authenticators/features/authn_oidc.feature

```diff
@@ -175,8 +175,8 @@ Feature: Users can authneticate with OIDC authenticator
   Scenario: Performance test
     And I fetch an ID Token for username "alice" and password "alice"
     When I authenticate 1000 times in 10 threads via OIDC with id token
-    Then The "max" response time should be less than "1" seconds
-    And The "avg" response time should be less than "0.25" seconds
```

Comment:
How are these changes related to broader PR?  Would it make more sense to introduce a new PR for these OIDC performance adjustments?  PRs really should be specific bits of tightly related functionality.

Reply:
Is this related to failing tests? If so, can we try to understand why it's failing rather than just adjust performance tests to potentially accommodate an actual code related problem?